target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -135,6 +135,11 @@ zyxel,keenetic-extra-ii)
 	set_wifi_led "$boardname:green:wifi"
 	ucidef_set_led_switch "internet" "internet" "$boardname:green:internet" "switch0" "0x01"
 	;;
+zyxel,keenetic-4g)
+	set_wifi_led "$boardname:green:wifi"
+	ucidef_set_led_switch "internet" "internet" "$boardname:green:internet" "switch0" "0x01"
+	;;
+esac
```

Comment:
One esac will be enough. Sorting.
Wifi LED should be implemented in device tree.